README.md

```diff
@@ -20,43 +21,133 @@ This stack will deploy a php application on X Amazon EC2 instances behind an ALB
   * **Elasticache** (optional): Amazon Elasticache (redis)
   * **S3 bucket** (optional): public medias bucket
 
-> **Pipeline** The pipeline contains a manual approval between terraform plan and terraform apply.
-> That means if you trigger a terraform plan, to apply it, you have to go on terraform apply job
-> and click on the `+` button to trigger it.
-
 # Requirements
 
 In order to run this task, couple elements are required within the infrastructure:
 
-* Having a VPC with private & public subnets containing a bastion server that can access instances by SSH
-* Having an S3 bucket for terraform remote states
-* Having an S3 bucket for magento code WITH versioning enable
+  * Having a VPC with private & public subnets [Here](https://docs.aws.amazon.com/vpc/latest/userguide/getting-started-ipv4.html#getting-started-create-vpc)
+  * Having a bastion server to run Ansible like described [Here](https://docs.cycloid.io/advanced-guide/ansible-integration.html#standard-usage)
+  * Having an S3 bucket to store Terraform remote states [Here](https://docs.aws.amazon.com/quickstarts/latest/s3backup/step-1-create-bucket.html)
+  * Having an S3 bucket for website code **WITH** versioning enable [Here](https://docs.aws.amazon.com/quickstarts/latest/s3backup/step-1-create-bucket.html)
 
 
 # Details
 
-**Pipeline**
-
-  * Run packer to build debian Amazon AMI with nginx, php-fpm and telegraph metrics (port 9100 prometheus format).
-
-**Terraform**
+## Pipeline
 
-Create:
-
-  * ALB (loadbalancer)
-  * ASG with launch_template of fronts
-  * RDS (optional)
-  * Elasticache (optional)
-  * S3 bucket (optional) + Iam
+> **Note** The pipeline contains a manual approval between terraform plan and terraform apply.
+> That means if you trigger a terraform plan, to apply it, you have to go on terraform apply job
+> and click on the `+` button to trigger it.
 
-**Ansible**
+<img src="docs/pipeline.png" width="800">
+
+**Jobs description**
+
+  * `build-worker-ami` : Build Front Amazon image AMI using Packer and Ansible.
+  * `build-application`: Runs the appropriate php/composer commands and build a release file of the code to put it on S3.
+  * `unittest`: Dummy job meant to eventually be replaced by proper tests or removed.
+  * `terraform-plan`: Terraform job that will simply make a plan of the stack.
+  * `terraform-apply`: Terraform job similar to the plan one, but will actually create/update everything that needs to. Please see the plan diff for a better understanding.
+  * `application-deployment`: Simply trigger a deployment using Ansible of the last version of the code on existing instances.
+  * `functional-tests`: Dummy job meant to eventually be replaced by proper functional or removed.
+  * `terraform-destroy`: :warning: Terraform job meant to destroy the whole stack - **NO CONFIRMATION ASKED**. If triggered, the full project **WILL** be destroyed. Use with caution.
```

Comment:
Shall we specify that it's not displayed in the pipeline screenshot, but is on a dedicated pipeline tab?

Reply:
I think that's ok if this job is little bit harder to found :p